app/src/main/java/com/github/se/icebreakrr/model/message/MeetingRequestViewModel.kt

```diff
@@ -96,60 +94,46 @@ class MeetingRequestViewModel(
    *
    * @param newToken the current token of the user to which we want to send the message
    */
-  fun onLocalTokenChange(newToken: String) {
+  fun setTargetToken(newToken: String) {
```

Comment:
Nice refactor. This naming is more accurate now!

Reply:
Thank you ! It was indeed confusing